R/methods.R

```diff
@@ -40,6 +29,22 @@ format.lint <- function(x, ..., width = getOption("lintr.format_width")) {
   )
 }
 
+build_line_ref <- function(x) {
+  line_ref <- paste0(
+    x$filename, ":",
+    as.character(x$line_number), ":",
+    as.character(x$column_number)
+  )
+
+  create_clickable_link <- cli::ansi_has_hyperlink_support()
+  if (create_clickable_link) {
+    line_ref <- paste0("{.path ", line_ref, "}")
```

Comment:
This looks brittle to me. How will this handle closing curly braces in filenames?

Reply:
Seems like this works with this approach cc0cb5e

Although I just noticed that this works

```r
x <- "file{e.R"
cli::format_inline("{.path {x}}") |> cat()
```

While this doesn't work 

```r
cli::format_inline("{.path file{e.R}")
```